### DISCUSSION TITLE
Cant Find Source File

### DISCUSSION THREAD

**Discussion by VolcanicGameStudios** [community/community#86761] [Programming Help] [answered]
### Body

I currently am working on making a game engine and I made a folder called events inside of a folder called Volcanic and I am trying to get a header file inside of the events folder to get a header from the volcanic folder and whenever I do #include "Volcanic/Core.h" it says source file not found.
![Screenshot 2024-01-12 201014](https://github.com/community/community/assets/128656773/cca5a75a-94b3-43d9-bc97-1aa7805f0627)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by kmadaminjanova0508** [comment, marked as answer]
Good time of your day @BurritoWizard 
I think you are encountering an issue with including header files from different directories in your game engine project in that case you should check path of your project When including a header file from a different directory, you need to specify the relative path from the current file to the target header file. For example, if your folder structure looks like this:
      - GameEngine
     - Volcanic
       - Core.h
     - Events
       - EventHeader.h
   
   You would need to include "Core.h" in "EventHeader.h" using the following relative path:
      #include "../Volcanic/Core.h"
   
OR If you are using an IDE or build system, make sure that the include directories are set up correctly. You may need to specify the include directories in your project settings or build configuration so that the compiler can find the header files in different folders :) I think it should help you in that case 

Good luck on your further projects

**Reply by VolcanicGameStudios** [reply]
Hello, Thank you for the reply. It still doesn't work unfortunately. And also it wouldn't work when it was in the same folder. And I have recently implemented premake5

**Reply by VolcanicGameStudios** [reply]
Nvm thanks to you I fixed it
